plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/review-template/hooks.tsx

```diff
@@ -153,39 +152,22 @@ export const useCommentQueryArgs = ( { postId }: { postId: number } ) => {
 };
 
 /**
- * Generate a tree structure of comment IDs from a list of comment entities. The
- * children of each comment are obtained from `_embedded`.
+ * Generate a tree structure of comment IDs from a list of comment entities.
+ * For product reviews, this is a flat list as reviews don't support threading.
  */
 export const useCommentTree = (
```

Comment:
Not a big deal, but what do you think about renaming this hook to something like `useCommentList()` or even `useReviews()`? If we are no longer rendering a tree, it might be good to avoid all mentions of it? :thinking: